rocketchat_API/APISections/base.py

```diff
@@ -83,6 +83,12 @@ def call_api_get(self, method, api_path=None, **kwargs):
         )
 
     def call_api_post(self, method, files=None, use_json=None, **kwargs):
+        if "text" in kwargs:
```

Comment:
Could you please extract this into a separate function that sanitizes the text and add a comment/docstring with what the purpose of it is.
It'll make it easier to keep track of it in the future.
I'd also like to see this tested if possible. 
Eg: in https://github.com/jadolg/rocketchat_API/blob/b4c4c8819bc3e52013ef0f176101aacd090daeaf/tests/test_chat.py#L14 include some of these characters and check that the posted message has them by retrieving it.
Or create a new test for this purpose altogether.